charts/fleet-agent/fleet-agent/0.3.500/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+annotations:
```

Comment:
```bash
arvindiyengar: ~/Rancher/charts/src/github.com/rancher/charts 
$ tree | grep -Ev "\.yaml$|\.tpl$|\.txt$|\.md$|\.ps1$|\.json$|\.tgz$|\.svg$|\.yml$"
.
├── CNAME
├── Makefile
├── assets
│   ├── fleet
│   ├── fleet-agent
│   ├── fleet-crd
│   ├── logos
│   ├── longhorn
│   ├── rancher-alerting-drivers
│   ├── rancher-backup
│   ├── rancher-backup-crd
│   ├── rancher-cis-benchmark
│   ├── rancher-external-ip-webhook
│   ├── rancher-gatekeeper
│   ├── rancher-grafana
│   ├── rancher-istio
│   ├── rancher-istio-1.8
│   ├── rancher-istio-1.9
│   ├── rancher-kiali-server
│   ├── rancher-kube-state-metrics
│   ├── rancher-logging
│   ├── rancher-monitoring
│   ├── rancher-node-exporter
│   ├── rancher-operator
│   ├── rancher-operator-crd
│   ├── rancher-prom2teams
│   ├── rancher-prometheus-adapter
│   ├── rancher-pushprox
│   ├── rancher-sachet
│   ├── rancher-tracing
│   ├── rancher-vsphere-cpi
│   ├── rancher-vsphere-csi
│   ├── rancher-webhook
│   ├── rancher-windows-exporter
│   ├── rancher-wins-upgrader
│   └── rio
├── bin
│   └── charts-build-scripts
├── charts
│   ├── fleet
│   │   └── fleet
│   │       ├── 0.3.000
│   │       │   ├── charts
│   │       │   │   └── gitjob
│   │       │   │       ├── templates
│   │       │   ├── templates
│   │       ├── 0.3.100
│   │       │   ├── charts
│   │       │   │   └── gitjob
│   │       │   │       ├── templates
│   │       │   ├── templates
│   │       ├── 0.3.200
│   │       │   ├── charts
│   │       │   │   └── gitjob
│   │       │   │       ├── templates
│   │       │   ├── templates
│   │       ├── 0.3.300
│   │       │   ├── charts
│   │       │   │   └── gitjob
│   │       │   │       ├── templates
│   │       │   ├── templates
│   │       ├── 0.3.400
│   │       │   ├── charts
│   │       │   │   └── gitjob
│   │       │   │       ├── templates
│   │       │   ├── templates
│   │       └── 0.3.500
│   │           ├── charts
│   │           │   └── gitjob
│   │           │       ├── templates
│   │           ├── templates
│   ├── fleet-agent
│   │   └── fleet-agent
│   │       ├── 0.3.000
│   │       │   ├── templates
│   │       ├── 0.3.100
│   │       │   ├── templates
│   │       ├── 0.3.200
│   │       │   ├── templates
│   │       ├── 0.3.300
│   │       │   ├── templates
│   │       ├── 0.3.400
│   │       │   ├── templates
│   │       └── 0.3.500
│   │           ├── templates
│   ├── fleet-crd
│   │   └── fleet-crd
│   │       ├── 0.3.000
│   │       │   └── templates
│   │       ├── 0.3.100
│   │       │   └── templates
│   │       ├── 0.3.200
│   │       │   └── templates
│   │       ├── 0.3.300
│   │       │   └── templates
│   │       ├── 0.3.400
│   │       │   └── templates
│   │       └── 0.3.500
│   │           └── templates
│   ├── longhorn
│   │   ├── longhorn
│   │   │   ├── 1.0.200
│   │   │   │   ├── templates
│   │   │   ├── 1.0.201
│   │   │   │   ├── templates
│   │   │   ├── 1.1.000
│   │   │   │   ├── templates
│   │   │   ├── 1.1.001
│   │   │   │   ├── templates
│   │   │   └── 1.1.100
│   │   │       ├── templates
│   │   └── longhorn-crd
│   │       ├── 1.0.200
│   │       │   └── templates
│   │       ├── 1.0.201
│   │       │   └── templates
│   │       ├── 1.1.000
│   │       │   └── templates
│   │       ├── 1.1.001
│   │       │   └── templates
│   │       └── 1.1.100
│   │           └── templates
│   ├── rancher-alerting-drivers
│   │   └── rancher-alerting-drivers
│   │       └── 1.0.100
│   │           ├── charts
│   │           │   ├── prom2teams
│   │           │   │   ├── files
│   │           │   │   │   └── teams.j2
│   │           │   │   ├── templates
│   │           │   └── sachet
│   │           │       ├── files
│   │           │       │   └── template.tmpl
│   │           │       ├── templates
│   │           ├── templates
│   ├── rancher-backup
│   │   ├── rancher-backup
│   │   │   ├── 1.0.200
│   │   │   │   ├── templates
│   │   │   ├── 1.0.201
│   │   │   │   ├── templates
│   │   │   ├── 1.0.300
│   │   │   │   ├── templates
│   │   │   ├── 1.0.301
│   │   │   │   ├── templates
│   │   │   └── 1.0.400
│   │   │       ├── files
│   │   │       │   └── default-resourceset-contents
│   │   │       ├── templates
│   │   └── rancher-backup-crd
│   │       ├── 1.0.200
│   │       │   └── templates
│   │       ├── 1.0.201
│   │       │   └── templates
│   │       ├── 1.0.300
│   │       │   └── templates
│   │       └── 1.0.301
│   │           └── templates
│   ├── rancher-backup-crd
│   │   └── rancher-backup-crd
│   │       └── 1.0.400
│   │           └── templates
│   ├── rancher-cis-benchmark
│   │   ├── rancher-cis-benchmark
│   │   │   ├── 1.0.100
│   │   │   │   ├── templates
│   │   │   ├── 1.0.200
│   │   │   │   ├── templates
│   │   │   ├── 1.0.300
│   │   │   │   ├── templates
│   │   │   ├── 1.0.301
│   │   │   │   ├── templates
│   │   │   └── 1.0.400
│   │   │       ├── templates
│   │   └── rancher-cis-benchmark-crd
│   │       ├── 1.0.100
│   │       │   └── templates
│   │       ├── 1.0.200
│   │       │   └── templates
│   │       ├── 1.0.300
│   │       │   └── templates
│   │       ├── 1.0.301
│   │       │   └── templates
│   │       └── 1.0.400
│   │           └── templates
│   ├── rancher-external-ip-webhook
│   │   └── rancher-external-ip-webhook
│   │       ├── 0.1.400
│   │       │   ├── templates
│   │       │   ├── tests
│   │       ├── 0.1.600
│   │       │   ├── templates
│   │       │   ├── tests
│   │       └── 0.1.601
│   │           ├── templates
│   │           ├── tests
│   ├── rancher-gatekeeper
│   │   ├── rancher-gatekeeper
│   │   │   ├── 3.1.100
│   │   │   │   ├── templates
│   │   │   ├── 3.1.101
│   │   │   │   ├── templates
│   │   │   ├── 3.2.101
│   │   │   │   ├── templates
│   │   │   ├── 3.3.000
│   │   │   │   ├── templates
│   │   │   └── 3.3.001
│   │   │       ├── templates
│   │   └── rancher-gatekeeper-crd
│   │       ├── 3.1.100
│   │       │   └── templates
│   │       ├── 3.1.101
│   │       │   └── templates
│   │       ├── 3.2.101
│   │       │   ├── crd-manifest
│   │       │   ├── templates
│   │       ├── 3.3.000
│   │       │   ├── crd-manifest
│   │       │   ├── templates
│   │       └── 3.3.001
│   │           ├── crd-manifest
│   │           ├── templates
│   ├── rancher-grafana
│   │   └── rancher-grafana
│   │       └── 6.6.401
│   │           ├── dashboards
│   │           ├── templates
│   │           │   └── tests
│   ├── rancher-istio
│   │   └── rancher-istio
│   │       ├── 1.7.100
│   │       │   ├── charts
│   │       │   │   └── rancher-kiali-server
│   │       │   │       ├── templates
│   │       │   │       │   ├── dashboards
│   │       │   ├── configs
│   │       │   ├── requirements.lock
│   │       │   ├── samples
│   │       │   ├── templates
│   │       ├── 1.7.300
│   │       │   ├── charts
│   │       │   │   └── rancher-kiali-server
│   │       │   │       ├── templates
│   │       │   │       │   ├── dashboards
│   │       │   ├── configs
│   │       │   ├── requirements.lock
│   │       │   ├── samples
│   │       │   ├── templates
│   │       ├── 1.7.301
│   │       │   ├── charts
│   │       │   │   ├── rancher-kiali-server
│   │       │   │   │   ├── templates
│   │       │   │   │   │   ├── dashboards
│   │       │   │   └── rancher-tracing
│   │       │   │       ├── templates
│   │       │   ├── configs
│   │       │   ├── requirements.lock
│   │       │   ├── samples
│   │       │   ├── templates
│   │       └── 1.8.300
│   │           ├── charts
│   │           │   ├── kiali
│   │           │   │   ├── templates
│   │           │   │   │   ├── dashboards
│   │           │   └── tracing
│   │           │       ├── templates
│   │           ├── configs
│   │           ├── samples
│   │           ├── templates
│   ├── rancher-istio-1.8
│   │   └── rancher-istio
│   │       └── 1.8.500
│   │           ├── charts
│   │           │   ├── kiali
│   │           │   │   ├── templates
│   │           │   │   │   ├── dashboards
│   │           │   └── tracing
│   │           │       ├── templates
│   │           ├── configs
│   │           ├── samples
│   │           ├── templates
│   ├── rancher-istio-1.9
│   │   └── rancher-istio
│   │       └── 1.9.300
│   │           ├── charts
│   │           │   ├── kiali
│   │           │   │   ├── templates
│   │           │   │   │   ├── dashboards
│   │           │   └── tracing
│   │           │       ├── templates
│   │           ├── configs
│   │           ├── samples
│   │           ├── templates
│   ├── rancher-kiali-server
│   │   ├── rancher-kiali-server
│   │   │   ├── 1.23.001
│   │   │   │   ├── templates
│   │   │   │   │   ├── dashboards
│   │   │   ├── 1.24.001
│   │   │   │   ├── templates
│   │   │   │   │   ├── dashboards
│   │   │   ├── 1.24.003
│   │   │   │   ├── templates
│   │   │   │   │   ├── dashboards
│   │   │   ├── 1.29.000
│   │   │   │   ├── templates
│   │   │   │   │   ├── dashboards
│   │   │   └── 1.32.100
│   │   │       ├── templates
│   │   │       │   ├── dashboards
│   │   └── rancher-kiali-server-crd
│   │       ├── 1.23.001
│   │       │   └── templates
│   │       ├── 1.24.001
│   │       │   └── templates
│   │       ├── 1.24.003
│   │       │   └── templates
│   │       ├── 1.29.000
│   │       │   └── templates
│   │       └── 1.32.100
│   │           └── templates
│   ├── rancher-kube-state-metrics
│   │   └── rancher-kube-state-metrics
│   │       └── 2.13.101
│   │           ├── LICENSE
│   │           ├── OWNERS
│   │           ├── templates
│   ├── rancher-logging
│   │   ├── rancher-logging
│   │   │   ├── 3.6.000
│   │   │   │   ├── templates
│   │   │   │   │   ├── loggings
│   │   │   │   │   │   ├── eks
│   │   │   │   │   │   ├── k3s
│   │   │   │   │   │   ├── rke
│   │   │   │   │   │   ├── rke2
│   │   │   │   │   │   └── root
│   │   │   ├── 3.6.001
│   │   │   │   ├── templates
│   │   │   │   │   ├── loggings
│   │   │   │   │   │   ├── eks
│   │   │   │   │   │   ├── k3s
│   │   │   │   │   │   ├── rke
│   │   │   │   │   │   ├── rke2
│   │   │   │   │   │   └── root
│   │   │   ├── 3.8.201
│   │   │   │   ├── templates
│   │   │   │   │   ├── loggings
│   │   │   │   │   │   ├── aks
│   │   │   │   │   │   ├── eks
│   │   │   │   │   │   ├── gke
│   │   │   │   │   │   ├── k3s
│   │   │   │   │   │   ├── rke
│   │   │   │   │   │   ├── rke2
│   │   │   │   │   │   └── root
│   │   │   ├── 3.9.000
│   │   │   │   ├── templates
│   │   │   │   │   ├── loggings
│   │   │   │   │   │   ├── aks
│   │   │   │   │   │   ├── eks
│   │   │   │   │   │   ├── gke
│   │   │   │   │   │   ├── k3s
│   │   │   │   │   │   ├── rke
│   │   │   │   │   │   ├── rke2
│   │   │   │   │   │   └── root
│   │   │   └── 3.9.400
│   │   │       ├── templates
│   │   │       │   ├── loggings
│   │   │       │   │   ├── aks
│   │   │       │   │   ├── eks
│   │   │       │   │   ├── gke
│   │   │       │   │   ├── k3s
│   │   │       │   │   ├── rke
│   │   │       │   │   ├── rke2
│   │   │       │   │   └── root
│   │   └── rancher-logging-crd
│   │       ├── 3.6.000
│   │       │   └── templates
│   │       ├── 3.6.001
│   │       │   └── templates
│   │       ├── 3.8.201
│   │       │   └── templates
│   │       ├── 3.9.000
│   │       │   └── templates
│   │       └── 3.9.400
│   │           └── templates
│   ├── rancher-monitoring
│   │   ├── rancher-monitoring
│   │   │   ├── 14.5.100
│   │   │   │   ├── charts
│   │   │   │   │   ├── grafana
│   │   │   │   │   │   ├── dashboards
│   │   │   │   │   │   ├── templates
│   │   │   │   │   │   │   └── tests
│   │   │   │   │   ├── k3sServer
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── kube-state-metrics
│   │   │   │   │   │   ├── LICENSE
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── kubeAdmControllerManager
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── kubeAdmEtcd
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── kubeAdmProxy
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── kubeAdmScheduler
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── prometheus-adapter
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── prometheus-node-exporter
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── rke2ControllerManager
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── rke2Etcd
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── rke2Proxy
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── rke2Scheduler
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── rkeControllerManager
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── rkeEtcd
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── rkeProxy
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── rkeScheduler
│   │   │   │   │   │   ├── templates
│   │   │   │   │   └── windowsExporter
│   │   │   │   │       ├── scripts
│   │   │   │   │       ├── templates
│   │   │   │   ├── files
│   │   │   │   │   ├── ingress-nginx
│   │   │   │   │   └── rancher
│   │   │   │   │       ├── cluster
│   │   │   │   │       ├── etcd
│   │   │   │   │       ├── home
│   │   │   │   │       ├── k8s
│   │   │   │   │       ├── linux
│   │   │   │   │       ├── nodes
│   │   │   │   │       ├── pods
│   │   │   │   │       ├── windows
│   │   │   │   │       └── workloads
│   │   │   │   ├── templates
│   │   │   │   │   ├── alertmanager
│   │   │   │   │   ├── exporters
│   │   │   │   │   │   ├── core-dns
│   │   │   │   │   │   ├── kube-api-server
│   │   │   │   │   │   ├── kube-controller-manager
│   │   │   │   │   │   ├── kube-dns
│   │   │   │   │   │   ├── kube-etcd
│   │   │   │   │   │   ├── kube-proxy
│   │   │   │   │   │   ├── kube-scheduler
│   │   │   │   │   │   ├── kube-state-metrics
│   │   │   │   │   │   ├── kubelet
│   │   │   │   │   │   └── node-exporter
│   │   │   │   │   ├── grafana
│   │   │   │   │   │   ├── dashboards
│   │   │   │   │   │   ├── dashboards-1.14
│   │   │   │   │   ├── prometheus
│   │   │   │   │   │   ├── rules
│   │   │   │   │   │   ├── rules-1.14
│   │   │   │   │   ├── prometheus-operator
│   │   │   │   │   │   ├── admission-webhooks
│   │   │   │   │   │   │   ├── job-patch
│   │   │   │   │   ├── rancher-monitoring
│   │   │   │   │   │   ├── dashboards
│   │   │   │   │   │   │   ├── addons
│   │   │   │   │   │   │   └── rancher
│   │   │   │   │   │   ├── exporters
│   │   │   │   │   │   │   └── ingress-nginx
│   │   │   ├── 9.4.200
│   │   │   │   ├── charts
│   │   │   │   │   ├── grafana
│   │   │   │   │   │   ├── dashboards
│   │   │   │   │   │   ├── templates
│   │   │   │   │   │   │   └── tests
│   │   │   │   │   ├── kube-state-metrics
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── prometheus-adapter
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── prometheus-node-exporter
│   │   │   │   │   │   ├── templates
│   │   │   │   │   └── rancher-pushprox
│   │   │   │   │       ├── templates
│   │   │   │   ├── requirements.lock
│   │   │   │   ├── templates
│   │   │   │   │   ├── alertmanager
│   │   │   │   │   ├── exporters
│   │   │   │   │   │   ├── core-dns
│   │   │   │   │   │   ├── kube-api-server
│   │   │   │   │   │   ├── kube-controller-manager
│   │   │   │   │   │   ├── kube-dns
│   │   │   │   │   │   ├── kube-etcd
│   │   │   │   │   │   ├── kube-proxy
│   │   │   │   │   │   ├── kube-scheduler
│   │   │   │   │   │   ├── kube-state-metrics
│   │   │   │   │   │   ├── kubelet
│   │   │   │   │   │   └── node-exporter
│   │   │   │   │   ├── grafana
│   │   │   │   │   │   ├── dashboards
│   │   │   │   │   │   ├── dashboards-1.14
│   │   │   │   │   ├── prometheus
│   │   │   │   │   │   ├── rules
│   │   │   │   │   │   ├── rules-1.14
│   │   │   │   │   ├── prometheus-operator
│   │   │   │   │   │   ├── admission-webhooks
│   │   │   │   │   │   │   ├── job-patch
│   │   │   │   │   ├── rancher-monitoring
│   │   │   ├── 9.4.201
│   │   │   │   ├── charts
│   │   │   │   │   ├── grafana
│   │   │   │   │   │   ├── dashboards
│   │   │   │   │   │   ├── templates
│   │   │   │   │   │   │   └── tests
│   │   │   │   │   ├── kube-state-metrics
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── prometheus-adapter
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── prometheus-node-exporter
│   │   │   │   │   │   ├── templates
│   │   │   │   │   └── rancher-pushprox
│   │   │   │   │       ├── templates
│   │   │   │   ├── requirements.lock
│   │   │   │   ├── templates
│   │   │   │   │   ├── alertmanager
│   │   │   │   │   ├── exporters
│   │   │   │   │   │   ├── core-dns
│   │   │   │   │   │   ├── kube-api-server
│   │   │   │   │   │   ├── kube-controller-manager
│   │   │   │   │   │   ├── kube-dns
│   │   │   │   │   │   ├── kube-etcd
│   │   │   │   │   │   ├── kube-proxy
│   │   │   │   │   │   ├── kube-scheduler
│   │   │   │   │   │   ├── kube-state-metrics
│   │   │   │   │   │   ├── kubelet
│   │   │   │   │   │   └── node-exporter
│   │   │   │   │   ├── grafana
│   │   │   │   │   │   ├── dashboards
│   │   │   │   │   │   ├── dashboards-1.14
│   │   │   │   │   ├── prometheus
│   │   │   │   │   │   ├── rules
│   │   │   │   │   │   ├── rules-1.14
│   │   │   │   │   ├── prometheus-operator
│   │   │   │   │   │   ├── admission-webhooks
│   │   │   │   │   │   │   ├── job-patch
│   │   │   │   │   ├── rancher-monitoring
│   │   │   ├── 9.4.202
│   │   │   │   ├── charts
│   │   │   │   │   ├── grafana
│   │   │   │   │   │   ├── dashboards
│   │   │   │   │   │   ├── templates
│   │   │   │   │   │   │   └── tests
│   │   │   │   │   ├── kube-state-metrics
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── prometheus-adapter
│   │   │   │   │   │   ├── templates
│   │   │   │   │   ├── prometheus-node-exporter
│   │   │   │   │   │   ├── templates
│   │   │   │   │   └── rancher-pushprox
│   │   │   │   │       ├── templates
│   │   │   │   ├── requirements.lock
│   │   │   │   ├── templates
│   │   │   │   │   ├── alertmanager
│   │   │   │   │   ├── exporters
│   │   │   │   │   │   ├── core-dns
│   │   │   │   │   │   ├── kube-api-server
│   │   │   │   │   │   ├── kube-controller-manager
│   │   │   │   │   │   ├── kube-dns
│   │   │   │   │   │   ├── kube-etcd
│   │   │   │   │   │   ├── kube-proxy
│   │   │   │   │   │   ├── kube-scheduler
│   │   │   │   │   │   ├── kube-state-metrics
│   │   │   │   │   │   ├── kubelet
│   │   │   │   │   │   └── node-exporter
│   │   │   │   │   ├── grafana
│   │   │   │   │   │   ├── dashboards
│   │   │   │   │   │   ├── dashboards-1.14
│   │   │   │   │   ├── prometheus
│   │   │   │   │   │   ├── rules
│   │   │   │   │   │   ├── rules-1.14
│   │   │   │   │   ├── prometheus-operator
│   │   │   │   │   │   ├── admission-webhooks
│   │   │   │   │   │   │   ├── job-patch
│   │   │   │   │   ├── rancher-monitoring
│   │   │   └── 9.4.203
│   │   │       ├── charts
│   │   │       │   ├── grafana
│   │   │       │   │   ├── dashboards
│   │   │       │   │   ├── templates
│   │   │       │   │   │   └── tests
│   │   │       │   ├── k3sServer
│   │   │       │   │   ├── templates
│   │   │       │   ├── kube-state-metrics
│   │   │       │   │   ├── templates
│   │   │       │   ├── kubeAdmControllerManager
│   │   │       │   │   ├── templates
│   │   │       │   ├── kubeAdmEtcd
│   │   │       │   │   ├── templates
│   │   │       │   ├── kubeAdmProxy
│   │   │       │   │   ├── templates
│   │   │       │   ├── kubeAdmScheduler
│   │   │       │   │   ├── templates
│   │   │       │   ├── prometheus-adapter
│   │   │       │   │   ├── templates
│   │   │       │   ├── prometheus-node-exporter
│   │   │       │   │   ├── templates
│   │   │       │   ├── rke2ControllerManager
│   │   │       │   │   ├── templates
│   │   │       │   ├── rke2Etcd
│   │   │       │   │   ├── templates
│   │   │       │   ├── rke2Proxy
│   │   │       │   │   ├── templates
│   │   │       │   ├── rke2Scheduler
│   │   │       │   │   ├── templates
│   │   │       │   ├── rkeControllerManager
│   │   │       │   │   ├── templates
│   │   │       │   ├── rkeEtcd
│   │   │       │   │   ├── templates
│   │   │       │   ├── rkeProxy
│   │   │       │   │   ├── templates
│   │   │       │   └── rkeScheduler
│   │   │       │       ├── templates
│   │   │       ├── requirements.lock
│   │   │       ├── templates
│   │   │       │   ├── alertmanager
│   │   │       │   ├── exporters
│   │   │       │   │   ├── core-dns
│   │   │       │   │   ├── kube-api-server
│   │   │       │   │   ├── kube-controller-manager
│   │   │       │   │   ├── kube-dns
│   │   │       │   │   ├── kube-etcd
│   │   │       │   │   ├── kube-proxy
│   │   │       │   │   ├── kube-scheduler
│   │   │       │   │   ├── kube-state-metrics
│   │   │       │   │   ├── kubelet
│   │   │       │   │   └── node-exporter
│   │   │       │   ├── grafana
│   │   │       │   │   ├── dashboards
│   │   │       │   │   ├── dashboards-1.14
│   │   │       │   ├── prometheus
│   │   │       │   │   ├── rules
│   │   │       │   │   ├── rules-1.14
│   │   │       │   ├── prometheus-operator
│   │   │       │   │   ├── admission-webhooks
│   │   │       │   │   │   ├── job-patch
│   │   │       │   ├── rancher-monitoring
│   │   └── rancher-monitoring-crd
│   │       ├── 14.5.100
│   │       │   ├── crd-manifest
│   │       │   ├── templates
│   │       ├── 9.4.200
│   │       │   ├── crd-manifest
│   │       │   ├── templates
│   │       ├── 9.4.201
│   │       │   ├── crd-manifest
│   │       │   ├── templates
│   │       ├── 9.4.202
│   │       │   ├── crd-manifest
│   │       │   ├── templates
│   │       └── 9.4.203
│   │           ├── crd-manifest
│   │           ├── templates
│   ├── rancher-node-exporter
│   │   └── rancher-node-exporter
│   │       └── 1.16.201
│   │           ├── OWNERS
│   │           ├── ci
│   │           ├── templates
│   ├── rancher-operator
│   │   └── rancher-operator
│   │       ├── 0.1.000
│   │       │   ├── templates
│   │       ├── 0.1.100
│   │       │   ├── templates
│   │       ├── 0.1.200
│   │       │   ├── templates
│   │       ├── 0.1.300
│   │       │   ├── templates
│   │       └── 0.1.400
│   │           ├── templates
│   ├── rancher-operator-crd
│   │   └── rancher-operator-crd
│   │       ├── 0.1.000
│   │       │   └── templates
│   │       ├── 0.1.100
│   │       │   └── templates
│   │       ├── 0.1.200
│   │       │   └── templates
│   │       ├── 0.1.300
│   │       │   └── templates
│   │       └── 0.1.400
│   │           └── templates
│   ├── rancher-prom2teams
│   │   └── rancher-prom2teams
│   │       └── 0.2.000
│   │           ├── files
│   │           │   └── teams.j2
│   │           ├── templates
│   ├── rancher-prometheus-adapter
│   │   └── rancher-prometheus-adapter
│   │       └── 2.12.101
│   │           ├── ci
│   │           ├── templates
│   ├── rancher-pushprox
│   │   └── rancher-pushprox
│   │       ├── 0.1.0
│   │       │   ├── templates
│   │       ├── 0.1.1
│   │       │   ├── templates
│   │       ├── 0.1.2
│   │       │   ├── templates
│   │       ├── 0.1.201
│   │       │   ├── templates
│   │       └── 0.1.300
│   │           ├── templates
│   ├── rancher-sachet
│   │   └── rancher-sachet
│   │       └── 1.0.100
│   │           ├── files
│   │           │   └── template.tmpl
│   │           ├── templates
│   ├── rancher-tracing
│   │   └── rancher-tracing
│   │       ├── 1.20.001
│   │       │   ├── templates
│   │       ├── 1.20.002
│   │       │   ├── templates
│   │       └── 1.20.100
│   │           ├── templates
│   ├── rancher-vsphere-cpi
│   │   └── rancher-vsphere-cpi
│   │       └── 1.0.000
│   │           ├── templates
│   ├── rancher-vsphere-csi
│   │   └── rancher-vsphere-csi
│   │       └── 2.1.000
│   │           ├── templates
│   ├── rancher-webhook
│   │   └── rancher-webhook
│   │       ├── 0.1.0-beta500
│   │       │   ├── templates
│   │       ├── 0.1.0-beta700
│   │       │   ├── templates
│   │       ├── 0.1.0-beta900
│   │       │   ├── templates
│   │       ├── 0.1.0-beta901
│   │       │   ├── templates
│   │       └── 0.1.000
│   │           ├── templates
│   ├── rancher-windows-exporter
│   │   └── rancher-windows-exporter
│   │       └── 0.1.000
│   │           ├── scripts
│   │           ├── templates
│   ├── rancher-wins-upgrader
│   │   └── rancher-wins-upgrader
│   │       └── 0.0.100
│   │           ├── scripts
│   │           ├── templates
│   └── rio
│       └── rio
│           ├── 0.8.000
│           │   ├── templates
│           └── 0.8.001
│               ├── templates
├── index.html
└── scripts
    ├── pull-scripts
    ├── regenerate-assets
    └── version

748 directories, 3779 files
```

Reply:
Seems like we don't have any weird files like `.orig` either from what I can read, so LGTM here too